Library/Homebrew/cmd/update-report.rb

```diff
@@ -520,9 +526,8 @@ def migrate_formula_rename
       end
 
       begin
-        migrator = Migrator.new(f)
+        migrator = Migrator.new(f, force: true)
```

Comment:
Given the only thing the force does is skip the tap origin check and the purpose of this PR is to fix up those cases: I think so. I'm happy to remove it but we may need to be willing to quickly re-add it if it causes issues without. Similarly, I think there's a different between a manual `brew migrate` and something that's run automatically and the automatic one shouldn't be prompting for something that users will blindly run anyway.